docker-compose/docker-compose.yaml

```diff
@@ -1,5 +1,53 @@
 version: '2.4'
 services:
+  # Description: This container will migrate the Postgres `frontend`,
+  # `codeintel`, and `codeinsights` databases
+  #
+  # Disk: None
+  # Ports: None
+  #
+  # This container should start after the database instance starts, perform the migrations
+  # and then exit cleanly. After exiting the `frontend` service is allowed to start.
+  #
+  # If you're using a DB instance hosted outside of docker-compose, the environment variables
+  # for this container will need to be updated to reflect the new connection information.
+  migrator:
+    container_name: migrator
+    image: 'index.docker.io/sourcegraph/migrator:122149_2021-12-17_1f7179c'
+    cpus: 4
+    mem_limit: '8g'
+    command:
+      ["up"]
+    environment:
+      - PGHOST=pgsql
+      - PGPORT=5432
+      - PGUSER=sg
+      - PGPASSWORD=sg
+      - PGDATABASE=sg
+      - PGSSLMODE=disable
+
+      - CODEINTEL_PGHOST=codeintel-db
+      - CODEINTEL_PGPORT=5432
+      - CODEINTEL_PGUSER=sg
+      - CODEINTEL_PGPASSWORD=sg
+      - CODEINTEL_PGDATABASE=sg
+      - CODEINTEL_PGSSLMODE=disable
+
+      - CODEINSIGHTS_PGHOST=codeinsights-db
+      - CODEINSIGHTS_PGPORT=5432
+      - CODEINSIGHTS_PGUSER=postgres
+      - CODEINSIGHTS_PGPASSWORD=password
+      - CODEINSIGHTS_PGDATABASE=postgres
+      - CODEINSIGHTS_PGSSLMODE=disable
+    restart: "on-failure"
+    networks: 
+      - sourcegraph
+    depends_on:
+      pgsql:
+        condition: service_healthy
+      codeintel-db:
+        condition: service_healthy
```

Comment:
This should also depend on codeinsights-db, right?

Reply:
`codeinsights-db` unfortunately doesn't expose a healthcheck endpoint defined that we're able to check against so we can't wait for the `service_healthy` condition. I looked into adding it but because it's leveraging TimescaleDB I'm a little weary on how to check if it's actually "ready".

Reply:
Boo! That ruins the totally-awesome suggestion I was going to make for the docs PR as an alternative to the manual docker run command. You can run `docker-compose up migrator` and it only starts up the migrator container and associated dependencies.

But, without codeinsights-db, the migrator can't run so my suggestion won't work 😢